jonathanmusila/Questioner-Django
# #165659394 code-climate integration

#### What does this PR do?
- add code climate badge

#### Description of Task to be completed?
 - [x] add code climate badge
- [x] add Travis builds badge

### How should this be manually tested?
- go to the repo 
`https://github.com/jonathanmusila/Questioner-Django`
- open the develop brach 
- scroll down to see readme

#### What are the relevant pivotal tracker stories?
#165659394
